CHANGELOG.md

```diff
@@ -50,7 +78,11 @@ We check progress in the next reconciliation loop. ([#362](https://github.com/gi
 - Separate podsecuritypolicy.
 - Security context in deployment spec with non-root user.
 
-[Unreleased]: https://github.com/giantswarm/chart-operator/compare/v0.12.1..HEAD
+[Unreleased]: https://github.com/giantswarm/chart-operator/compare/v0.13.0..HEAD
+[v0.13.0]: https://github.com/giantswarm/chart-operator/releases/tag/v0.13.0
+[v0.12.4]: https://github.com/giantswarm/chart-operator/releases/tag/v0.12.4
+[v0.12.3]: https://github.com/giantswarm/chart-operator/releases/tag/v0.12.3
+[v0.12.2]: https://github.com/giantswarm/chart-operator/releases/tag/v0.12.2
```

Comment:
We discussed here, https://github.com/giantswarm/chart-operator/pull/440#discussion_r416479762 We will make it another PR in master branch, since this is a PR for catching up with the current master branch.